### DISCUSSION TITLE
How can I set the icon in the TitleBar on GTK/Skia on Windows?

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#14908] [Q&A] [answered]
I'm running my Uno 5 application on Windows, using Skia/GTK. How can I change the default icon of the application? I'm not sure if it's Windows-specific as I'm running full-screen on Linux so it's not an issue there. This is the icon I'd like to change:
![image](https://github.com/unoplatform/uno/assets/407941/fd5e98ec-af1f-4ea1-9300-d8f10f4e693e)

I tried to set `ExtendViewIntoTitleBar`, but it didn't seem to make a difference, or I'm not sure how to use it or what to expect.

**Comment by jeromelaban** [comment, marked as answer]
This icon is coming from the imported Package.appxmanifest file from the Windows file. It's set through:

https://github.com/unoplatform/uno/blob/604120780556100fb2264ac90d5a57e237aa70c0/src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs#L98

When [this line is set](https://github.com/unoplatform/uno.templates/blob/e71e64a75e32dd213ff3cb0ffc11fd87bde8be09/src/Uno.Templates/content/unoapp/MyExtensionsApp.1.Skia.Gtk/MyExtensionsApp.1.Skia.Gtk.csproj#L9) in the template.

**Reply by HakanL** [reply]
Hmm, that makes sense, but I have those set, however it's set like this in my GTK head:
```
<ItemGroup Condition="exists('..\DMXCore100.Windows')">
   <EmbeddedResource Include="..\DMXCore100.Windows\Package.appxmanifest" LogicalName="Package.appxmanifest" />
</ItemGroup>
```
and then in the linked appxmanifest I have this:
```
  <Properties>
	<DisplayName>DMX Core 100</DisplayName>
	<PublisherDisplayName>DMX Core</PublisherDisplayName>
	<Logo>Images\StoreLogo.png</Logo>
  </Properties>
```
And that image exists (under the Windows head project), but it's not the one that is displayed (screenshot above). I tried to also create an Images/StoreLogo.png file under the GTK head, still didn't work. What am I missing?

**Reply by jeromelaban** [reply]
Take a look at the logs for the app, it may indicate what's happening. Otherwise, something else may not be working and we'll need to open an issue with a repro.

**Reply by HakanL** [reply]
Yup: `Unable to find icon file [C:\Projects\DMXCore\100\Software\DMXCore100.Skia.Gtk\bin\Debug\net8.0\Images\StoreLogo.png] specified in the Package.appxmanifest file.`. So I added the logo to the GTK head, but I forgot to set the build action to `Content`. After that it worked, thanks for pointing me in the right direction!
![image](https://github.com/unoplatform/uno/assets/407941/7dd9d943-a86a-4756-8580-8fdca513147b)
